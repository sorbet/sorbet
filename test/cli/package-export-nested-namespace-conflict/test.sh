cd test/cli/package-export-nested-namespace-conflict || exit 1

../../../main/sorbet --silence-dev-message --stripe-packages --max-threads=0 . 2>&1

