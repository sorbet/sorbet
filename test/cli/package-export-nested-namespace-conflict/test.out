parent/__package.rb:4: Conflicting import sources for `Parent::Child` https://srb.help/3714
     4 |  import Parent::Child
                 ^^^^^^^^^^^^^
    parent/__package.rb:6: Conflict from
     6 |  export_for_test Parent
                          ^^^^^^
Errors: 1
